### DISCUSSION TITLE
Orderd list with letters

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/ods-mkdocs-material#51] [Domande] [answered]
Buonasera a tutti! Ho letto [questo paragrafo](https://squidfunk.github.io/mkdocs-material/reference/lists/?h=order#using-ordered-lists) relativo alle liste ordinate e mi chiedevo: sarebbe possibile generare una lista ordinata che abbia le lettere al posto dei numeri (di default)?

es.

a. ciao
b. ciao
c. ciao

**Comment by aborruso** [comment]
Ciao @dennisangemi non credo. Salvo scriverlo in HTML

**Reply by dennisangemi** [reply]
Naggia! pensavo si potesse fare dato che per le liste ordinate indentate vengono utilizzate le lettere anche da material 4 mkdocs. Se ci fosse un modo per "modificare la priorità" sarebbe perfetto. Se non riesco a trovare nulla utilizzerò html. Grazie mille

![image](https://user-images.githubusercontent.com/77018886/151415487-3c9f20d4-7197-4a2f-ae60-3d43b9ecb8e1.png)

**Comment by gbvitrano** [comment, marked as answer]
@dennisangemi lo vuoi modificare su l'intero sito o su una pagina in particolare?
in entrambi i casi va modificata la regola css dell'elenco puntato `.md-typeset ul`
in quest'[esempio ](https://opendatasicilia.github.io/OpenDataSicilia-per-il-Coronavirus/test/test_03/) applico la regola ad una sola pagina, aggiungendo  `.md-typeset ul  {list-style-type: lower-latin!important;}`
Se vuoi applicare la regola a tutto il sito,  basta aggiungere la stessa regola al file extra.css (o al tuo file css personalizzato)
Esistono tanti stili, nella demo ti ho messo il link al CSS list-style-type Propert

![2022-01-29_23h03_51](https://user-images.githubusercontent.com/20488693/151678994-6eb5c7f1-8601-4575-88ed-2a5c9d86f448.png)

**Reply by dennisangemi** [reply]
Wow gianni! Grazie mille, in effetti mi serviva proprio per l'intero sito perché si tratta di tantissimi file con la stessa struttura. Proverò appena posso e segnerò come la discussione come risolta. Grazie ancora

**Reply by dennisangemi** [reply]
Funziona tutto! Grazie mille Gianni
